devito/types/sparse.py

```diff
@@ -608,7 +613,7 @@ def _index_matrix(self, offset):
 
         return index_matrix, points
 
-    def _interpolation_indices(self, variables, offset=0):
+    def _interpolation_indices(self, variables, offset=0, field_offset=0):
```

Comment:
[PR off topic ...]
while I appreciate the need for `field_offset`, I do not understand what is `offset` still there for. Did we not agree that it even breaks mpi?